plutarch-test/src/Plutarch/Test.hs

```diff
@@ -69,26 +86,17 @@ pshouldBe x y = do
 passert :: ClosedTerm a -> Expectation
 passert p = p #@?= pcon PTrue
 
+{- Asserts the term to be false -}
```

Comment:
Shouldn't this be `{- | Asserts the term to be false -}`, so Haddock picks it up?